eventually/comment/models.py

```diff
@@ -5,16 +5,16 @@
 This module implements class that represents the comment entity.
```

Comment:
maybe it is a good idea to refactor existing models in the other branches?

Reply:
done